_positions/front-end-designer.md

```diff
@@ -34,6 +34,7 @@ active: true
 -   Work with the 18F design team on what it means to “design in the open.”
 -   Contribute to and help maintain 18F’s [front end standards](https://pages.18f.gov/frontend/).
 
+##[Apply to the Front End Designer role](https://jobs.lever.co/18f/bcb9af9a-460a-48d4-bfaf-a62defe6fb4d/apply)
```

Comment:
@rooneywp Mind moving this to the end of the page instead for consistency with the others (ex: [Visual Designer](https://github.com/18F/joining-18f/blob/61c589ed4664eccb27500e674d78f8f39f50291f/_positions/visual-designer.md))?